modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustServerCodegen.java

```diff
@@ -1561,4 +1562,31 @@ private void processParam(CodegenParameter param, CodegenOperation op) {
             param.vendorExtensions.put("example", (example != null) ? "Some(" + example + ")" : "None");
         }
     }
+
+    @Override
+    public void postProcessFile(File file, String fileType) {
+        if (file == null) {
+            return;
+        }
+
+        String commandPrefix = System.getenv("RUST_SERVER_POST_PROCESS_FILE");
```

Comment:
What about renaming `RUST_SERVER_POST_PROCESS_FILE` as `RUST_POST_PROCESS_FILE` to conform to other environment variable naming `{lang}_POST_PROCESS_FILE` (e.g. CPP_POST_PROCESS_FILE)?

Reply:
And what about adding something similar to the following in `processOpts`?

https://github.com/OpenAPITools/openapi-generator/commit/f647b2f24be707acb5fc23d3093449fddad373c4#diff-7946cad868d081bf8881db60979b628bR256-R259

Reply:
Done - thanks :smile: